OAuth: Add support for empty scopes (#32129)
grafana/grafana: docs/sources/auth/generic-oauth.md

client_id = YOUR_APP_CLIENT_ID
client_secret = YOUR_APP_CLIENT_SECRET
scopes =

> empty_scopes =

Reviewer: Shouldn't this be either `false` or `true`? `empty_scopes` is supposed to be a boolean, not a (potentially empty) string.

`tls_skip_verify_insecure` controls whether a client verifies the server's certificate chain and host name. If it is true, then SSL/TLS accepts any certificate presented by the server and any host name in that certificate. _You should only use this for testing_, because this mode leaves SSL/TLS susceptible to man-in-the-middle attacks.

> Set `no_scope_parameter` to true to use an empty scope in your authentication process. By default Grafana will use `user:email` as scope.

Reviewer 2: ```suggestion
Set `no_scope_parameter` to true to use an empty scope during authentication. By default, Grafana will use `user:email` as scope.
```
